app/org/sagebionetworks/bridge/play/controllers/TaskController.java

```diff
@@ -30,14 +34,28 @@ public void setTaskService(TaskService taskService) {
         this.taskService = taskService;
     }
     
-    public Result getTasks(String untilString) throws Exception {
+    public Result getTasks(String untilString, String offset, String daysAhead) throws Exception {
```

Comment:
This isn't hard to add after this refactor, but would require additional tests, etc and this needs to get out. I've made a JIRA for this.